utils/find-overlay-dependencies.sh

```diff
@@ -68,16 +68,22 @@ for sdk in ${(k)SDKS}; do
   DEPENDS_ON=()
   printf "%s:\n\t" "$sdk"
   deps=$(echo "@import $1;" | xcrun -sdk $sdk clang -arch $arch -x objective-c - -M -fmodules 2>/dev/null)
+  if [[ $? != 0 ]]; then
+    # Don't `continue` here because we want to clear out the SDK line if it exists
```

Comment:
I'm worried about putting it in the diff because I think we'll fire-and-forget on this tool.

(If there's an error in the clang line, it'd be nice to see that too. It's just annoying to print stderr to the console but capture stdout in a variable; I don't remember how to do it properly.